src/Microsoft.Data.SqlClient/tests/ManualTests/TracingTests/DiagnosticTest.cs

```diff
@@ -64,20 +62,19 @@ public void ExecuteScalarErrorTest()
                         cmd.CommandText = "select 1 / 0;";
 
                         conn.Open();
-
-                        try { var output = cmd.ExecuteScalar(); }
+                        try
+                        { var output = cmd.ExecuteScalar(); }
                         catch { }
```

Comment:
Could be better if we catch the expected error.

Reply:
we do not want to catch  the exception it just swallows  the exceptions.